packages/uniforms-material/__tests__/DateField.tsx

```diff
@@ -132,18 +132,32 @@ test('<DateField> - renders a Input with correct value (default)', () => {
   expect(wrapper.find(Input).prop('value')).toBe('');
 });
 
-test('<DateField> - renders a Input with correct value (model)', () => {
+const testDateType = (type: AllowedDateTypes = 'datetime-local') => {
```

Comment:
`test.each` would be better here.

Reply:
@radekmie Updated the PR, please review again. (Sorry if I missed something, found a break at my work to do this)